### PR TITLE
mobile-friendly and datatables for views

### PR DESCRIPTION
**Pull request by galdazbiz**
It's a first approximation, it can be improved but it's a good start for a mobile-friendly forum :)
I added it into a new folder so the people can choose, but I think the idea would be to merge at some point.

### CODE REVIEW

**Comment by marktopper** [comment]
Thanks, but those are not actually used in the system.
We would be open for a solution where the user can choose to use your set of views or have them working with the current set of views.

Thanks.

**Comment by galdazbiz** [comment]
Any dev or contributor can help with linking the views to the routes/controller? I'm better at frontend than programming.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "devdojo/chatter",
+    "name": "tronne/chatter",
```
Comment (marktopper) [inline composer.json:2]: Don't change the package name.

**Review by marktopper — CHANGES_REQUESTED** [review]

**Comment by galdazbiz** [comment]
I cancel this to continue with my develop rather than improving the original code because it take too much time

**Pull request closed without merging**